packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -160,9 +160,20 @@ typedef NestedScrollViewHeaderSliversBuilder = List<Widget> Function(BuildContex
 ///
 /// ### Stretching [SliverAppBar]s
 ///
-// See https://github.com/flutter/flutter/issues/54059
 /// Currently, [NestedScrollView] does not support stretching the outer
```

Comment:
This still says stretch is not supported.

Reply:
Done